packages/react-next/src/components/Slider/useSlider.ts

```diff
@@ -250,15 +335,22 @@ export const useSlider: (props: ISliderProps, ref: React.Ref<HTMLDivElement>) =>
   const containerProps = {
     className: classNames.container,
   };
+  const sliderLineProps = {
+    ref: sliderLine,
+    className: classNames.line,
+  };
 
   return {
     root: rootProps,
     label: labelProps,
     sliderBox: sliderBoxProps,
     container: containerProps,
     valueLabel: valueLabelProps,
-    classNames,
-    sliderLine,
-    value,
+    thumb: thumbProps,
+    zeroTick: zeroTickProps,
+    activeTrack: trackActiveProps,
+    topInactiveTrack: trackTopInactiveProps,
+    bottomInactiveTrack: trackBottomInactiveProps,
+    sliderLine: sliderLineProps,
```

Comment:
i see a long way to go from here to use `compose`. but i don't mind to take this step :)